### ISSUE TITLE
Nothing is loading

### ISSUE THREAD

**Issue by robbinonline** [MarkusTheOrt/F1TVUnofficial#3]
Is there a change in the API? Nothing is loading here.

**Comment by MarkusTheOrt** [comment]
Yes, the API URLs changed. I haven’t had any time to change it tho.

**Comment by robbinonline** [comment]
Ah I found is it just replacing all:
https://f1tv.formula.com/api/

into
https://f1tv-api.formula1.com/agl/1.0/gbr/en/all_devices/global/

**Comment by MarkusTheOrt** [comment]
yes, you are right. However I am heading of to work in approx. 20 Minutes so I'll push something tomorrow. Just a heads up that the current (new) master branch does not yet support live sessions.

**Comment by MarkusTheOrt** [comment]
Seems like after a more thorough look at it, it is a bit more work than just replacing the urls. This might take some time to adjust for.

**Comment by robbinonline** [comment]
No need to hurry thanks to covid-19 :(

**Comment by luisfosoares** [comment]
Hello, any news?

**Comment by MarkusTheOrt** [comment]
Nothing as of now. I haven't done any work towards this. Most likely because if I'd want to continue (which is probably happening sometime in the future) it'll be a full rewrite (since they made so huge API changes) and I haven't really bothered with looking into it.
